app/admin.go

```diff
@@ -218,3 +218,13 @@ func (a *App) TestEmail(userId string, cfg *model.Config) *model.AppError {
 
 	return nil
 }
+
+// ServerBusyStateChanged is called when a CLUSTER_EVENT_BUSY_STATE_CHANGED is received.
+func (a *App) ServerBusyStateChanged(sbs *model.ServerBusyState) {
+	a.Srv.Busy.ClusterEventChanged(sbs)
+	if sbs.Busy {
+		mlog.Warn("server busy state activitated via cluster event - non-critical services disabled", mlog.Int64("seconds", sbs.Expires))
```

Comment:
I'd change the name of the mlog field to "expires_sec"

Reply:
Thanks @reflog.  Logging field renamed via https://github.com/mattermost/mattermost-server/pull/13267/commits/e4aae17f678d2cbece5ded814ce8ab57dddaa800